content/source/layouts/docs-frontpage.erb

```diff
@@ -1,10 +1,7 @@
 <% wrap_layout :inner do %>
   <% content_for :sidebar do %>
-    <h4><a href="/docs/index.html">Terraform Docs</a></h4>
 
-    <%= partial(
-                "layouts/otherdocs", :locals => { :skip => "none" }
-        ).sub(%r{^ *<h4>Other Docs</h4> *$}, '').gsub(/class="back" /, '') %>
```

Comment:
We used to remove `class="back"` and now we don't. Is that change intentional? I can't find a CSS definition for `.back` anywhere…

Reply:
Yeahhhh I kind of dropped .back from the styling when I made the nav sections collapsable/expandable, so it's inert now. Could bring it back sometime, but as a reader I always found it inscrutable.